package.json

```diff
@@ -8,14 +8,15 @@
 		"node": ">=4"
 	},
 	"scripts": {
-		"test": "xo && tsc --project types && nyc ava",
+		"test": "xo && tsc --project types && flow --max-warnings 0 && nyc ava",
 		"bench": "matcha benchmark.js",
 		"coveralls": "nyc report --reporter=text-lcov | coveralls"
 	},
 	"files": [
 		"index.js",
 		"templates.js",
-		"types/index.d.ts"
+		"types/index.d.ts",
+		"index.js.flow"
```

Comment:
Can you move the file to the `types` directory, same with the Flow test.

Reply:
I wish this was possible :/

Originally, I had wanted this PR to have a folder structure like so:

```
types
├── flow
│   ├── .flowconfig
│   ├── index.js.flow
│   └── test.js
└── typescript
    ├── index.d.ts
    ├── test.ts
    └── tsconfig.json
```

There are some limitations on the Flow side of things though. 

1. The `.flowconfig` needs to be within the root project folder that has `node_modules` in it. We could get around this by having a script like this though:

```json
{
  "scripts": {
    "flow": "cd types/flow && flow --max-warnings 0"
  }
}
```

2. The `index.js.flow` file needs to be in the same folder of the file it is shadowing. Because there is no build process in your project (where every file ends up in `lib` for example), I don't think there is any way to get around this, so the only way I can think of to allow this is to introduce a build process in your project which creates a `lib` folder that has `index.js`, `index.d.ts`, and a`index.js.flow` inside it. I don't know if you'd be willing to do something like that for this change though.

@jamiebuilds or anyone else please correct me if you have any better suggestions.

Reply:
Also note: there is an issue open [here](https://github.com/facebook/flow/issues/2675) for Flow that would allow you to have a field in `package.json` to point to the flow libdef, but currently there is nothing equivalent to TypeScript's `types`/`typings` field :(

Reply:
I see. Ok, what you have is good for now then. We can improve it in the future when/if Flow adds the package.json field.